man/journalctl.xml

```diff
@@ -93,6 +94,13 @@
     <para>When outputting to a tty, lines are colored according to priority: lines of level ERROR and higher
     are colored red; lines of level NOTICE and higher are highlighted; lines of level DEBUG are colored
     lighter grey; other lines are displayed normally.</para>
+
+    <para>To write entries <emphasis>to</emphasis> the journal, a few methods may be used. In general, output
+    from systemd units is automatically connected to the journal, see
+    <citerefentry><refentrytitle>systemd-journald.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
```

Comment:
I think we should explicitly mention here that stderr and syslog() output are picked up by default, as that's probably what most people will be wondering about